packages/amesos2/src/KLU2/Include/klu2_internal.h

```diff
@@ -198,67 +198,6 @@ size_t KLU_kernel_factor            /* 0 if failure, size of LU if OK */
     KLU_common<Entry, Int> *Common  /* the control input/output structure */
 ) ;
 
-template <typename Entry, typename Int>
```

Comment:
Note for the reviewer(s): Here I deleted the forward declarations because there is no ETI for these. I guess in the very first version it was here a header file, and `klu2.hpp` was actually a `.cpp` file. And then it was changed to become templated.